client/packages/core/src/Reactor.js

```diff
@@ -969,6 +969,13 @@ export default class Reactor {
     safeSubs.forEach(({ eventId, q }) => {
       this._trySendAuthed(eventId, { op: "add-query", q });
     });
+
+    Object.values(this.queryOnceDfds)
+      .flat()
+      .forEach(({ eventId, q }) => {
+        this._trySendAuthed(eventId, { op: "add-query", q });
```

Comment:
Afaik there is now one weird invariant, where we could send an add-query twice. i.e `q` exists both in queryOnceDfds _and_ safeSubs. 

Is there a way we can collect both `subs` and `queryOnceDfds`, and return a single result of `qs`?

--- 

(Alternatively we can leave this as is, but it may lead to add-query-ok returning twice, or something like that)